iac/terraform/aws/cognito/policy.tf

```diff
@@ -24,6 +24,15 @@ locals {
         ]
         resources = [var.dynamodb_table_arn]
       }
+    ],
+    cognito_policy = [
+      {
+        sid = "CognitoPolicy"
+        actions = [
+          "cognito-idp:AdminUpdateUserAttributes"
+        ]
+        resources = ["*"]
```

Comment:
Why is this *?

Reply:
this is bad practice. but ran into circular dependency. 
`user pool` > `user pool trigger lambda` > `lambda policy` > `user pool`

Not sure whats the solution here

Reply:
made the policy more specific